src/components/entity/credentialItem/credentialItem.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <v-expansion-panel class="disabled">
+    <v-expansion-panel-header>
+      <slot name="header"></slot>
+    </v-expansion-panel-header>
+    <v-expansion-panel-content
+      :style="dropdownDivider ? 'background-color:lightgray' : ''"
```

Comment:
I'd much rather we use classes that inline styling. That way we only have to modify the styles in a few places.